src/tasks/gitea/gitea-drone-oauth.test.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai'
+import {
+  DroneSecret,
+  getGiteaAuthorizationHeaderCookies,
+  GiteaDroneError,
+  isSecretValid,
+  getCookieValue,
+} from './gitea-drone-oauth'
+import { OAuth2Application } from '@redkubes/gitea-client-node'
+import sinon from 'sinon'
+import axios from 'axios'
+
+describe('Gitea-Drone: Validate Secrets', () => {
```

Comment:
this whole suite can go, as you don't need to create that function `isSecretValid` in the first place as I tried to tell you